c/tskit/tables.h

```diff
@@ -961,19 +961,61 @@ on and may change arbitrarily between versions.
 */
 void tsk_individual_table_print_state(const tsk_individual_table_t *self, FILE *out);
 
-/** @} */
+/**
+@brief Replace this table's data by copying from a set of column arrays
 
-/* Undocumented methods */
+@rst
+Clears the data columns of this table and then copies column data from the specified
+set of arrays. The supplied arrays should all contain data on the same number of rows.
+The metadata schema is not affected.
+@endrst
 
+@param self A pointer to a tsk_individual_table_t object.
+@param num_rows The number of rows to copy from the specifed arrays
+@param flags A pointer to the tsk_flags_t array of flags to be copied
```

Comment:
Nitpicking here, but this isn't a pointer to an array, it's a pointer or an array (a pointer to an array might be interpreted as tsk_flags_t **). I see we're using this language in the ``add_row`` methods for things like location though. Hmm.

I would say something like ``The array of location values to be copied`` maybe? The type information is redundant and leaving out the pointer language makes it a bit clearer maybe? But, we should probably be consistent, so maybe ``The array of double location values to be copied.``?